cli/azd/internal/appdetect/appdetect.go

```diff
@@ -103,6 +103,21 @@ func (f Dependency) IsWebUIFramework() bool {
 	return false
 }
 
+type RawDependency struct {
```

Comment:
If it's easier, we can just return the RawProject pom.xml representation and let the "app generation" part decide what to do with that information. I think we want to avoid having additional schema translations